packages/web-core/src/services/ProcessService.ts

```diff
@@ -532,12 +532,18 @@ export class ProcessService {
     return WebAuthnService.doesBrowserSupportConditionalUI();
   }
 
-  async loginWithPasskeyChallenge(challenge: string): Promise<Result<ProcessResponse, CorbadoError>> {
+  async loginWithPasskeyChallenge(
+    challenge: string,
+    onLoginStart?: () => void,
```

Comment:
onLoginStart might be a misleading name (it indicates that this callback is executed once a login is started => in fact we only run the callback AFTER the interaction with the authenticator)

Maybe we name it onAuthenticatorCompleted?